src/hotspot/cpu/x86/universalNativeInvoker_x86.cpp

```diff
@@ -30,6 +30,7 @@
 #define __ _masm->
 
 void ProgrammableInvoker::Generator::generate() {
+#ifdef _LP64
```

Comment:
Instead of introducing ifdefs, I'd prefer to see `universalNativeInvoker_x86.cpp` split into `universalNativeInvoker_x86_64.cpp` (current code) and `universalNativeInvoker_x86_32.cpp` (populated with dummy impelmentations).

Reply:
Yeah, makes sense. The old code had ifdefs, but splitting seems cleaner to me as well.